### DISCUSSION TITLE
two values in "alert"

### DISCUSSION THREAD

**Discussion by Art-Che** [AutomaApp/automa#1030] [Q&A]
Hello!
Your app saves me at least 60 minutes every working day!

I need to take text values from a page, from several places, and display them in a message in the active tab. How can I do that?
![image](https://user-images.githubusercontent.com/120256667/209257235-2fe088f1-17c1-4e4b-949d-bb3f93d060e8.png)

I tried to do so, but an error occurs.
`alert(automaRefData('variables', 'name' + 'variables', 'phone'))
automaNextBlock()`

**Comment by Art-Che** [comment]
Addition:
i did like this:
```
var a = automaRefData('variables', 'name');
var b = automaRefData('variables', 'phone');
alert(a + ' ' + b);
automaNextBlock()
```
it turns out what I need. But for some reason there are two identical messages.

**Reply by Kholid060** [reply]
Can you show the variables in the log of the workflow?

**Reply by Art-Che** [reply]
after I cleared the cache in automata and chrome the error disappeared.
